### DISCUSSION TITLE
Feature request: cancel bundle requests

### DISCUSSION THREAD

**Discussion by thegostep** [flashbots/pm#26] [searchers]
When submitting a bundle to the flashbots relay, a unique hash is generated. It would be convenient to be able to cancel a bundle request if no longer desired.

**Comment by thegostep** [comment]
Should this be implemented at the relay level or at the mev-geth level?

Would it require a new rpc method?

**Comment by obadiaa** [comment]
More color on the usefulness of this feature:
Searchers submit bundles based on a state of the system. For searchers who use Flashbots for _non-atomic trading_, the state of the system they're trading against changes within the ~13s that it takes on average to create an Ethereum block.  

Without a bundle cancellation feature, such searchers are subject to adverse selection. To remediate this currently, they bid miners less than they would if cancellations existed to factor risks of adverse selection. 

The lack of cancellation arguably gives this type of trading on flashbots a higher barrier to entry (because sizing up your bid appropriately is no easy task) and potentially makes miners less money (since non-atomic searchers end up bidding lower than they would otherwise).

While this might be resolved with a cancellation feature, one concern to have with such a feature is the load it would have on our systems.

**Comment by Birdza87** [comment]
.
